data/movements/scripts/walkback.lua

```diff
@@ -1,6 +1,6 @@
 function onStepIn(creature, item, position, fromPosition)
 	if item.uid > 0 and item.uid <= 65535 then
-		creature:teleportTo(fromPosition, false)
+		return false
 	end
 	return true
```

Comment:
Just read through the code and from what I understood, if uid is >65535 the unique id was generated, not set through the map/actions.

`addThing` can return 0 if the item is removed, though I'm not sure if this is ever possible, it's better to be on the safe side.